ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/monitor-telemetry-postcard.cfg

```diff
@@ -0,0 +1,47 @@
+!RANCID-CONTENT-TYPE: arista
+!
+transceiver qsfp default-mode 4x10G
+!
+hostname monitor-telemetry-postcard
+!
+no enable password
+no aaa root
+!
+interface Management1
+   description oob_management
+   vrf MGMT
+   ip address 10.73.255.122/24
+!
+monitor telemetry postcard policy
+   no disabled
+   ingress collection gre source 10.3.3.3 destination 10.3.3.4 version 2
+   ingress sample rate 16384
+   ingress sample tcp-udp-checksum 65000
```

Comment:
Can we check if this is a valid config? On my EOS device, this was not a valid config
```
coi414(config-tele-postcard-policy)#   ingress sample tcp-udp-checksum 65000
% Invalid input
```
however this was:
```
coi414(config-tele-postcard-policy)# ingress sample tcp-udp-checksum value 65000 mask ?
  0x0-0xffff  16 bit hex mask for TCP/UDP or IP ID with atmost 2 unset bits
```
```
coi414(config)#sh ver
Arista CCS-720DF-48Y-2F
```

Reply:
Fixed.